.github/issue_template.yml

```diff
@@ -0,0 +1,25 @@
+---
+name: New Feature (with prototype)
+about: Use this template new features with links to prototypes.
+# These aren't used at the moment.
+# See: https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/manually-creating-a-single-issue-template-for-your-repository
+title: null
+labels: null
+assignees: null
+---
+<sup>ℹ️ [Figma (root)][figma] | [Sitemap][sitemap]</sup>
+<sup>_This initial comment is collaborative and open to modification by all._</sup>
+
```

Comment:
Maybe add a `## Description` section to delineate from the `<sup>` section?

Reply:
```suggestion

## Description
```